packages/babel-types/src/validators/isType.ts

```diff
@@ -17,6 +17,9 @@ export default function isType(
 export default function isType(nodeType: string, targetType: string): boolean {
   if (nodeType === targetType) return true;
 
+  // if nodeType is null or undefined, it can't be an alias of targetType
+  if (!nodeType) return false;
```

Comment:
```suggestion
  if (nodeType != null) return false;
```
Since we just need nullish check.

Can you also mark the `nodeType` parameter as `string | null | undefined`?

Reply:
Currently it seems that `""` can also be skipped, which is fine.
Also I don't think we should allow a null to be passed in, this should only happen in unexpected cases.